### DISCUSSION TITLE
xcodebuild fails with version 1.1.0

### DISCUSSION THREAD

**Discussion by everuribe** [pointfreeco/swift-case-paths#131] [Q&A]
**Describe the bug**
We get this error when running `xcodebuild` operations. We specifically pinned all our dependencies and only updated `swift-case-paths` to `1.1.0` and the error occurred in the package resolution phase. Could there be some versioning issue causing this?

```
 The following build commands failed:
  	ComputeTargetDependencyGraph
  (1 failure)
```

**Environment**
  - swift-case-paths 1.1.0
  - Xcode 15.0.1
  - OS: macOS 14.0 (arm64)

**Additional context**
Add any more context about the problem here.

**Comment by stephencelis** [comment]
@everuribe Can you paste the full output and share your Package.swift file or Xcode package configuration? There's not enough to go off of what you've currently shared. It's possible you just need to reset your package dependencies, re-resolve them, or you may just have conflicting versions specified in your package.

Since this doesn't appear to be a bug in the library I'm going to convert it to a Q&A discussion instead.

**Reply by everuribe** [reply]
Hey @stephencelis it looks like https://github.com/pointfreeco/swift-composable-architecture/pull/2577 resolves the problem. Thanks

**Reply by everuribe** [reply]
Actually no we are still getting issues with package resolution.

**Reply by stephencelis** [reply]
@everuribe Can you pin TCA to 1.4.2 and provide as much detail as possible in what the issue is? Or can you share a repro that we can work with?

**Comment by donnywdavis** [comment]
Hey. I just ran across this discussion as I was having the exact same error. Just wanted to share what worked for me in case others run across this as well. Everything worked locally, it was just on CI that we were having the problem. It is nothing related to the package but is only an issue when using the `xcodebuild` command and having to manually approve macros before it will compile. Since you can't do that on CI, that is what is throwing the error. Using the `-skipMacroValidation` compiler flag works to resolve the issue.

Obviously, these should be used with caution and only when using packages that you trust, but wanted to share in case it helps anyone else. 🙂
